include/platform/mir/graphics/display_configuration.h

```diff
@@ -185,6 +189,7 @@ struct UserDisplayConfigurationOutput
     MirOutputGammaSupported const& gamma_supported;
     std::vector<uint8_t const> const& edid;
     mir::optional_value<geometry::Size>& custom_logical_size;
+    std::string const name = "OUT-" + std::to_string(id.as_value());
```

Comment:
This should be a const reference to the underlying `DisplayConfigurationOutput` (like the other members)

Reply:
I think it may be necessary to not have this be a const reference since it is generating the string upon construction. If this is a reference to something temporary, it cannot compile.

Reply:
It shouldn't be a reference to a temporary. It should be a reference to `DisplayConfigurationOutput::name`